data/countries_names.txt

```diff
@@ -52682,49 +52682,78 @@
   uk = Ісфахан, Шираз, Ахваз
 
 [Iraq_North Description]
-  en = Mosul, Erbil, Kirkuk
-  ar = اربيل, كركوك
-  az = Mosul, Ərbil
-  da = Kirkuk
-  de = Mossul, Arbil, Kirkuk
-  el = Μοσούλη, Ερμπίλ, Κιρκούκ
-  fr = Mossoul, Erbil, Kirkouk
-  he = כירכוכ
-  ja = モースル, アルビール, キルクーク
-  pl = Mosul, Irbil, Kirkuk
-  ru = Мосул, Эрбиль, Киркук
-  sv = Kirkuk
-  tr = Musul, Erbil
-  uk = Мосул, Ербіль, Кіркук
+  en = Mosul, Erbil, Kirkuk, Tikrit
+  ar = الموصل، أربيل، كركوك، تكريت
+  az = Mosul, Ərbil, Kərkük, Tikrit
+  cs = Mosul, Erbíl, Kirkúk, Tikrít
+  da = Mosul, Erbil, Kirkuk, Tikrit
+  de = Mossul, Erbil, Kirkuk, Tikrit
+  el = Μοσούλη, Ερμπίλ, Κιρκούκ, Τικρίτ
+  es = Mosul, Erbil, Kirkuk, Tikrit
+  et = Mosul, Erbil, Kirkuk, Tikrit
+  fi = Mosul, Erbil, Kirkuk, Tikrit
+  fr = Mossoul, Erbil, Kirkouk, Tikrit
+  he = מוסול, ארביל, כירכוכ, תיכרית
+  hu = Moszul, Erbíl, Kirkuk, Tikrit
+  it = Mosul, Erbil, Kirkuk, Tikrit
+  ja = モスル, エルビル, キルクーク, ティクリート
+  ko = 모술, 에르빌, 키르쿠크, 티크리트
+  nl = Mosul, Erbil, Kirkuk, Tikrit
+  pl = Mosul, Irbil, Kirkuk, Tikrit
+  pt = Mossul, Erbil, Kirkuk, Tikrit
+  ro = Mosul, Erbil, Kirkuk, Tikrit
+  ru = Мосул, Эрбиль, Киркук, Тикрит
+  sk = Mosul, Erbíl, Kirkúk, Tikrít
+  sv = Mosul, Erbil, Kirkuk, Tikrit
+  th = โมซูล, เออร์บิล, เคอร์คุก, ติกริต
+  tr = Musul, Erbil, Kerkük, Tikrit
+  uk = Мосул, Ербіль, Кіркук, Тікріт
+  vi = Mosul, Erbil, Kirkuk, Tikrit
+  bg = Мосул, Ербил, Киркук, Тикрит
```

Comment:
Thanks! Please sort these translations by name (en should be the first one), and add
```
be = Масул, Эрбіл, Кірук, Цікрыт
```

Reply:
Added.